enterprise/frontend/src/embedding-sdk-bundle/components/public/StaticQuestion/StaticQuestion.tsx

```diff
@@ -113,28 +117,44 @@ const StaticQuestionInner = ({
       withDownloads={withDownloads}
     >
       {children ?? (
-        <FlexibleSizeComponent
-          width={width}
-          height={height}
-          className={className}
-          style={style}
-        >
-          <Stack gap="sm" w="100%" h="100%">
-            {title && <DefaultViewTitle title={title} />}
+        <FlexibleSizeComponent width={width} height={height} style={style}>
+          <Stack
+            className={cx(
+              hasTopBar
+                ? InteractiveQuestionS.ContainerTwoRows
+                : InteractiveQuestionS.ContainerSingleRow,
```

Comment:
I'd rather this be done in CSS.


```suggestion
              InteractiveQuestionS.Container,
```

then change the CSS like so
```css
.Container {
  display: grid;
  grid-template: 1fr / 1fr;
  grid-template-areas: "Main";
}

.Container:has(.TopBar) {
  display: grid;
  grid-template: auto 1fr / 1fr;
  grid-template-areas:
    "TopBar"
    "Main";
}
```